integration_tests/testsuite/driver.py

```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/python
+
+# Copyright 2016 Google Inc. All rights reserved.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+# http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import argparse
+import logging
+import sys
+
+import deploy_app
+import test_exception
+import test_logging
+import test_monitoring
+import test_root
+
+# TODO (nkubala): make this a configurable parameter from cloudbuild
+# required to be paired with '--no-deploy'
+DEFAULT_URL = 'https://nick-cloudbuild.appspot.com'
+
+
+def _main():
+    logging.getLogger().setLevel(logging.DEBUG)
+
+    parser = argparse.ArgumentParser()
+    parser.add_argument('--image', '-i', help='Newly-constructed base \
+                        image to build sample app on')
+
+    parser.add_argument('--directory', '-d', help='Root directory of \
+                        sample app')
+    # TODO (nkubala): potentially add support for multiple app directories
+    # to deploy
+    # parser.add_argument('--directory', '-d', help='Root directory of \
+    #                     sample app', action='append')
+    parser.add_argument('--no-deploy', action='store_true',
```

Comment:
Consider removing the double negative here so you can add the boolean opposites (e.g. --deploy) later if you want.  This would look like:

    parser.add_argument('--no-deploy', action='store_false', dest='deploy', help=...)
    ....
    if args.deploy: ...